ui/src/app/settings/views/Dhcp/DhcpList/DhcpList.js

```diff
@@ -2,17 +2,42 @@ import { useDispatch, useSelector } from "react-redux";
 import React, { useEffect } from "react";
 
 import actions from "app/settings/actions";
+import {
+  controller as controllerActions,
+  device as deviceActions,
+  machine as machineActions
+} from "app/base/actions";
+import {
+  controller as controllerSelectors,
+  device as deviceSelectors,
+  machine as machineSelectors
+} from "app/base/selectors";
 import Loader from "app/base/components/Loader";
 import selectors from "app/settings/selectors";
 
 // Temporarily create a item component, this will be replaced with a table.
-const DhcpListItem = ({ id, name, subnet }) => {
+const DhcpListItem = ({ id, name, node, subnet }) => {
   const subnetItem = useSelector(state =>
     selectors.subnet.getById(state, subnet)
   );
+  const controllerItem = useSelector(state =>
```

Comment:
I don't think we should be calling memoised selectors in our components (https://react-redux.js.org/next/api/hooks#using-memoizing-selectors), but we can fix that in a later branch.

Reply:
As discussed this is completely wrong apologies, we shouldn't **declare** memoised selectors in components, which we never do.